### PR TITLE
GEOPY-1194: Prepare release

### PR DESCRIPTION
**Pull request by domfournier**
**GEOPY-1194 - Release geoapps-0.11.0rc1**

### CODE REVIEW

File: pyproject.toml [hunk 1]
```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "geoapps"
-version = "0.11.0-beta.2"
+version = "0.11.0-rc1"
```
Comment (sebhmg) [inline pyproject.toml:3]: use format compliant with Semver
```suggestion
version = "0.11.0-rc.1"
```

File: docs/conf.py [hunk 2]
```diff
@@ -97,7 +97,7 @@
 # The short X.Y version.
 version = "0.11.0"
 # The full version, including alpha/beta/rc tags.
-release = "0.11.0-beta.2"
+release = "0.11.0-rc1"
```
Comment (sebhmg) [inline docs/conf.py:100]: use consistently a format compliant with Semver
```suggestion
release = "0.11.0-rc.1"
```

**Review by sebhmg — CHANGES_REQUESTED** [review]
version to be formatted according to Semver specifications

**Review by andrewg-mira — DISMISSED** [review]

**Comment by domfournier** [comment]
@sebhmg Somehow it's still requesting for changes. Can you review when you have the chance?

**Comment by domfournier** [comment]
Hang on one sec, need to update mira-simpeg version

**Review by sebhmg — DISMISSED** [review]

**Pull request merged**